scifin/timeseries/timeseries.py

```diff
@@ -2184,8 +2191,8 @@ def multi_plot(Series, figsize=(12,5), dpi=100):
             plt.plot(Series[i].data.index, Series[i].data.values)
         
     # Make it cute
-    title = "Multiplot of time series from " + str(min_date)[:10] \
-            + " to " + str(max_date)[:10]
+    if title is None:
+        title = "Multiplot of time series from " + str(min_date)[:10] + " to " + str(max_date)[:10]
```

Comment:
prefer using f-string instead of adding str together

Reply:
Thanks. Done.